src/library_fs.js

```diff
@@ -1136,6 +1138,14 @@ FS.staticInit();` +
       }
       stream.position = stream.stream_ops.llseek(stream, offset, whence);
       stream.ungotten = [];
+      try {
+        if (stream.path && FS.trackingDelegate['onSeekFile']) {
+          FS.trackingDelegate['onSeekFile'](stream.path, stream.position);
+
+        }
+      } catch(e) {
+        err("FS.trackingDelegate['onSeekFile']('"+stream.path+", streamPosition') threw an exception: " + e.message);
```

Comment:
It looks like it's actually injecting the value of `stream.path` into the error message here, so it would be good to do the same with `streamPosition`, rather than just printing a variable name. Same for the other new hook parameters below.

Reply:
Will do for the other hook parameters as well 👍